src/components/Banner/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from "./Banner";
+export { Label as BannerLabel } from "./types";
```

Comment:
Was wondering if maybe excluding all `index.ts` files from test coverage would make sense in order to not get the Codecov false positive?

Reply:
The report actually just revealed a bunch of places I still need to move exports (not sure why the linter didn't catch them). I'll push a change shortly.

Reply:
The changes look good! Thanks.